keyboards/hotdox/keymaps/bliss/keymap.c

```diff
@@ -0,0 +1,171 @@
+/* Copyright 2021 Benjamin Chausse
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+#define BASE 0 // default layer
+#define FUNC 1 // functions keys, numbpad, and arrows
```

Comment:
This will work the same as what you have now, but is easier to work with if you add or delete layers later - you won't have to re-number every layer.

```suggestion
enum layer_names {
    BASE, // default layer
    FUNC, // functions keys, numbpad, and arrows
};
```